### PR TITLE
docs: Fix markdown rendering

### PR DESCRIPTION
**Pull request by leader22**
Render `heading` properly. 👀

### CODE REVIEW

File: docs/CHEAT_SHEET.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ HTML attributes with the following syntax have a special meaning to Qwik.
 | `key:=JSON`         | State of the `Entity` identified by `key`, serialized into JSON.                                                        |
 | `:.=JSON`           | Component state, serialized into JSON.                                                                                  |
 
-### JSX Components
+### JSX Components
```
Comment (manucorporat) [inline docs/CHEAT_SHEET.md:21]: What is changing here?

Comment (leader22) [reply docs/CHEAT_SHEET.md:21]: TBH, I'm not sure what happening here. 😓 

But this change fixes [current](https://github.com/BuilderIO/qwik/blob/main/docs/CHEAT_SHEET.md) rendering output.

image

Comment (jimt) [reply docs/CHEAT_SHEET.md:21]: The original character following the `###` has its high bit set (`U+00a0`) when what is required is a simple ASCII space.

**Comment by leader22** [comment]
Closing since #133 fixed the same thing.

**Pull request closed without merging**